build-system/tasks/serve.js

```diff
@@ -193,19 +226,19 @@ async function serve() {
 
 /**
  * Starts a webserver at the repository root to serve built files.
- * @param {boolean=} lazyBuild
+ * @param {ServerOptionsDef=} serverOptions
  */
-async function doServe(lazyBuild = false) {
+async function doServe(serverOptions = {}) {
```

Comment:
Don't forget to update this call-site:

https://github.com/ampproject/amphtml/blob/9d4bf3117c3ab3c63e964407e3a546723189294a/build-system/tasks/default-task.js#L57